CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 ### Bug Fixes
 
 - Fixed a bug in Local Testing's implementation of LEFT ANTI and LEFT SEMI joins where rows with null values are dropped.
+- Fixed a bug when reading a filepath that does not exist, irrelevant error message is produced.
```

Comment:
```suggestion
- Improved the error message of `DataFrameReader` to raise `FileNotFound` error when reading a filepath that does not exist.
```
This should be an improvement instead of a bug fix.